h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -111,6 +111,7 @@ public void computeCrossValidation() {
    */
   @Override
   public void cv_computeAndSetOptimalParameters(ModelBuilder[] cvModelBuilders) {
+    if(_parms._max_runtime_secs != 0) _parms._max_runtime_secs = 0;
     if(_parms._lambda_search) {
```

Comment:
seb:  setting max_runtime_secs=0 will disable the time restriction.  I am confused to why you are setting it to 0 here.